crates/terminal_view/src/terminal_view.rs

```diff
@@ -945,6 +1094,8 @@ impl Render for TerminalView {
         let focused = self.focus_handle.is_focused(cx);
 
         div()
+            .occlude()
```

Comment:
@0xtimsb Curious why is that needed?
I seem to be able to click both the scrollbar area and the scrollbar thumb without it still; and without it, the drop is working again.